CesiumGltf/include/CesiumGltf/MetadataConversions.h

```diff
@@ -423,6 +500,35 @@ template <> struct MetadataConversions<double, float> {
   }
 };
 
+/**
+ * @brief Converts from std::string to a double.
+ */
+template <> struct MetadataConversions<double, std::string> {
+  /**
+   * Converts a std::string to a double. This assumes that the entire
+   * std::string represents the number, not just a part of it.
+   *
+   * This returns std::nullopt if no number is parsed from the string.
+   *
+   * @param from The std::string to parse from.
+   */
+  static std::optional<double> convert(const std::string& from) {
+    if (from.size() == 0) {
+      // Return early. Otherwise, empty strings will be parsed as 0, which is
+      // misleading.
+      return std::nullopt;
+    }
+
+    char* pLastUsed;
+    double parsedValue = std::strtod(from.c_str(), &pLastUsed);
+    if (pLastUsed == from.c_str() + from.size() && !std::isinf(parsedValue)) {
```

Comment:
```suggestion
    if (errno == 0 && pLastUsed == from.c_str() + from.size() && !std::isinf(parsedValue)) {
```

I think we missed talking about this line from the last review. Although looks like this isn't new code from this PR, would benefit from the same errno logic.